.github/workflows/docker.yml

```diff
@@ -123,14 +123,17 @@ jobs:
       - name: Install Docker Buildx
         uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # v3.0.0
 
-      - name: Login to the registry
-        run: >
-          echo ${{ secrets.GHCR_TOKEN }} |
-          docker login ${{ env.CONTAINER_NAMESPACE }}
-          --username abergs
-          --password-stdin
+      - name: Setup DCT
+        id: setup-dct
+        uses: bitwarden/gh-actions/setup-docker-trust@main
```

Comment:
Should this be snapshotted to a specific verison of the action?

Reply:
We pin our own actions to `main` to avoid the burden of updating them manually. We don't use tags, so we had problems using renovate for this task.